<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HdmiControlDialog</class>
 <widget class="QDialog" name="HdmiControlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>203</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="hdmiControlCheckBox">
     <property name="text">
      <string>HDMI Control</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="hdmiControlModeCheckBox">
     <property name="text">
      <string>HDMI Control Mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="arcCheckBox">
     <property name="text">
      <string>ARC</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Standby Pass Through</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="standbyPassThroughComboBox"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
